packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -687,6 +699,10 @@ export default class CFDStore extends BaseStore {
         this.real_swapfree_accounts_existing_data = real_swapfree_accounts_existing_data;
     }
 
+    getRealZerospreadAccountsExistingData(real_zerospread_accounts_existing_data) {
```

Comment:
```suggestion
    setRealZerospreadAccountsExistingData(real_zerospread_accounts_existing_data) {
```

It's actually saving data, not returning anything. So, `setRealZerospreadAccountsExistingData` would be a more appropriate name for the function.

Reply:
Since we're not going to have jurisdiction modal for zero_spread, I remove this value from it as it is not needed